datalad/interface/tests/test_run.py

```diff
@@ -695,22 +696,27 @@ def test_run_inputs_outputs(src, path):
 
     ds = install(path, source=src,
                  result_xfm='datasets', return_type='item-or-list')
-    assert_false(ds.repo.file_has_content("test-annex.dat"))
+    assert_false(ds.repo.file_has_content("input.dat"))
+    assert_false(ds.repo.file_has_content("extra-input.dat"))
 
-    # If we specify test-annex.dat as an input, it will be retrieved before the
-    # run.
-    ds.run("cat test-annex.dat test-annex.dat >doubled.dat",
-           inputs=["test-annex.dat"])
+    # The specified inputs and extra inputs will be retrieved before the run.
+    # (Use run_command() to access the extra_inputs argument.)
+    list(run_command("cat {inputs} {inputs} >doubled.dat",
+                     dataset=ds,
+                     inputs=["input.dat"], extra_inputs=["extra-input.dat"]))
```

Comment:
Sorry if I am missing obvious, but how do we know that extra inputs aren't part of `{inputs}` here? I was expecting to see the test of the expected output in doubled.dat

Reply:
> I was expecting to see the test of the expected output in doubled.dat

Oops, indeed, I meant to do that.  Thanks for spotting that.

Reply:
Why not just to use `ok_file_has_content('inputinput')` instead entire `with open ...` ?

Reply:
To me the explicit "not extra-input" displays the purpose of the check more clearly.